examples/angular/src/app/drawerWrapper/drawer.component.html

```diff
@@ -0,0 +1,16 @@
+<ds-drawer
+  [attr.footer-title]="footerTitle()"
+  [attr.heading]="drawerHeading()"
+  [attr.is-open]="isOpen"
+  analytics="true"
```

Comment:
I don't think our [`ds-drawer` component](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/components/web-components/ds-drawer/ds-drawer.tsx#L6-L20) has `analyticsOverrideAttrs`

Reply:
Oh I must've gotten confused because we're not even firing a ds-analytics-event from the ds-drawer. It is available on the Drawer component though. So maybe we can work to enable analytics on the web component as well.

Reply:
Opened https://jira.cms.gov/browse/WNMGDS-3158?atlLinkOrigin=c2xhY2staW50ZWdyYXRpb258aXNzdWU%3D